cli/tests/pcluster/config/test_validators.py

```diff
@@ -1827,13 +1827,6 @@ def test_dcv_enabled_validator(
                 supported_architectures=list(FSX_SUPPORTED_ARCHITECTURES_OSES.keys())
             ),
         ),
-        (
-            "arm64",
```

Comment:
We need to add this in the Supported combinations above

Reply:
It's there by nature, it's in `SUPPORTED_OSS`

Reply:
Sorry should have been clear, I meant we would want to add a new test case `("arm64", "centos7", None),` above